docs/robots/husky/tutorials_husky.mdx

```diff
@@ -4,7 +4,1535 @@ sidebar_position: 4
+| `/bluetooth_teleop`      | Publishes velocity commands from a joystick to `/twist_mux`                                                                                |
+| `/twist_mux`             | Takes in multiple sources of velocity commands, and prioritizes what actually gets sent to the controller                                  |
+| `/ekf_localization`      | Part of the robot localization package, more information regarding this package can be found at <http://wiki.ros.org/robot_localization>   |
+
+#### Topics
+
+You can view all topics that are active using `rostopic list`.
+The most important topics are summarized in the two tables below.
+
+##### General Topics
+
+| Topic                   | Message type             | Description                                                                                                                        |
+| :---------------------- | :----------------------- | :--------------------------------------------------------------------------------------------------------------------------------- |
+| `/bluetooth_teleop/joy` | `sensor_msgs/Joy`        | Receives joystick commands, echo this topic to verify your controller is publishing                                                |
+| `/tf`                   | `tf2_msgs/TFMessage`     | Transforms between coordinate frames, this should always be publishing, and hence a good topic to echo to test your ROS connection |
+| `/status`               | `husky_msgs/HuskyStatus` | Displays system status information                                                                                                 |
+| `/estop`                | `std_msgs/Bool`          | Displays the estop status                                                                                                          |
+| `/odometry/filtered`    | `nav_msgs/Odometry`      | The odometry estimate of the robot from \lstinline!/ekf_localization                                                               |
+
+##### Motion Topics    {#motion-topics}
+
+| Motion Topics                       | `twist_mux` Priority | Description                                                        |
+| :---------------------------------- | :------------------- | :----------------------------------------------------------------- |
+| `husky_velocity_controller/cmd_vel` | -                    | Receives motion commands from `twist_mux` based off their priority |
+| `joy_teleop/cmd_vel`                | 10                   | Joystick teleop input                                              |
+| `twist_marker_server/cmd_vel`       | 8                    | Interactive marker teleop input                                    |
+| `move_base/cmd_vel`                 | 2                    | Autonomous movement input, for the Husky navigation packages       |
+| `cmd_vel`                           | 1                    | Miscellaneous external input                                       |
```

Comment:
should a forward slash '/' prefix be added to all of these?

Reply:
Added the /